pandas/core/groupby/groupby.py

```diff
@@ -3228,15 +3231,17 @@ def ngroup(self, ascending: bool = True):
 
         Examples
         --------
-        >>> df = pd.DataFrame({"A": list("aaabba")})
+        >>> df = pd.DataFrame()
```

Comment:
Nit: Could you combine the DataFrame construction in 1 line (construct the DataFrame from dictionary?)